Tech-Journal-ConsoleApp/Tech-Journal-ConsoleApp/Program.cs

```diff
@@ -6,7 +6,19 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            var journal = new Journal();
+            var sendEmail = new SendEmail();
+            sendEmail.ValidateEmailSettings();
+            Console.WriteLine("Please enter your name:");
+            var userName = GetWelcomeMessage();
+            var entry = journal.JournalEntry();
+            sendEmail.Send(entry, userName);
+        }
+        public static string GetWelcomeMessage()
+        {
+            string userName = Console.ReadLine();
+            Console.WriteLine($"Hello {userName}! Today's Date is {DateTime.Now}");
```

Comment:
I think the welcome message can be it's own method, and getting the username can be it's own method.